Simplified Chinese Translation Typo Fix (1.18.2)

Cactusstudent opened the pull request:
As the PR title says, this is a PR focusing on the Typo Fixing. Include the file name, text and so on. This PR will update when I or the Chinese community find some typo. It maybe spend a long time. Hope this PR can be merged in the next **Release version**.

TODO List
- [ ] ~Optimize the text in each books~ Plan to do that in next couple of weeks, so it will not be contained in this PR
- [x] Fix Typo in zh_cn
- [x] Update based on the latest en_us

Cactusstudent committed:
Update zh_cn.json

add “cheese”
Cheese -> 奶酪

Co-Authored-By: ChuijkYahus <94828194+ChuijkYahus@users.noreply.github.com>

Cactusstudent committed:
Update Ency and Gadgets

Add the missing translation about new feature of the Slime Staff.

Cactusstudent committed:
Update language.lang in some of the books

Cactusstudent committed:
Translate the language.lang in some of the book

Based on the en_us

Cactusstudent committed:
Add translation of mighty

Cactusstudent committed:
Update material and you's book files

Cactusstudent committed:
Typo Fix

比啊 -> 便

Cactusstudent committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

Cactusstudent committed:
Update zh_cn based on en_us

Cactusstudent committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

Cactusstudent committed:
Update zh_cn

Cactusstudent committed:
Typo Fix

Cactusstudent committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

Cactusstudent committed:
Fix the wrong file name

Cactusstudent committed:
Fix the wrong file name

KnightMiner commented:
I have some more typo fixing to do in English which may impact you, notably I've been switch out the term "projectile damage" for "projectile power" as not all uses of the stst are strictly damage. It will be mostly encyclopedia that uses it. Not sure what terms you use, I'd make sure they match the translation for the tool stat in the main lang file

Cactusstudent commented:
Thanks for warning. Luckily, I have noticed this in the #5092 , and I changed the original term "`弹射物伤害`" to “`弹射物力量`” (additionally, `伤害` means **damage** in English, `力量` means **power** in English) to match the English term's change. But only in part of the zh_cn.json. The translation of guide books may still have tons of incorrect text. But I dont know if all of these should be changed, looks like some of the term use "伤害" is suitable, another seems more suitable to use "力量" (Such as the slings modifier). Maybe I do need a standard from you.

Cactusstudent committed:
材质 -> 纹理

Cactusstudent committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

Cactusstudent committed:
Translate

Translate the new command key based on the en_us.json

Cactusstudent committed:
牛奶 -> 奶

Because the vanilla term (Chinese) is “奶”

Cactusstudent committed:
Typo Fix

Cactusstudent committed:
Delete 使 to keep the fluency

Cactusstudent committed:
Optimize the text of the Tinkers' Gadgetry

Fix Typo
Change the expresssion

Cactusstudent committed:
Optimize the text of the Puny Smelting

Fix Typo
Transalted the text
Change the expression

Cactusstudent committed:
Small Fix

Delete the duplicate file and fix typo in Puny Smelting
Luckily, it doesn't a big matter.

Cactusstudent committed:
Finish the text correction of the Puny Smelting

Cactusstudent committed:
Optimize the expression of the Mighty Smelting

KnightMiner commented:
> Thanks for warning. Luckily, I have noticed this in the #5092 , and I changed the original term "`弹射物伤害`" to “`弹射物力量`” (additionally, `伤害` means **damage** in English, `力量` means **power** in English) to match the English term's change. But only in part of the zh_cn.json. The translation of guide books may still have tons of incorrect text. But I dont know if all of these should be changed, looks like some of the term use "伤害" is suitable, another seems more suitable to use "力量" (Such as the slings modifier). Maybe I do need a standard from you.

In updating the english text, everywhere we used "projectile damage" as a tool stat was more suited to "projectile power". The one case I saw of "projectile damage" that did not get changed was the description of projectile protection, as thats not the tool stat there

Cactusstudent committed:
Merge branch 'SlimeKnights:1.18.2' into 1.18.2

Cactusstudent committed:
Update zh_cn based on en_us

Cactusstudent committed:
Optimize the expression of the Mighty Smelting

Cactusstudent committed:
Typo Fixing of all the text (1st edition)

Cactusstudent committed:
Typo Fixing of the zh_cn.json

Cactusstudent committed:
Fix Typo

Cactusstudent committed:
Backport translation fix in 1.19

Cactusstudent committed:
Typo Fix Backport from 1.19

Cactusstudent committed:
Typo Fix Backport

Cactusstudent committed:
Translation fix backport

Cactusstudent committed:
武器 -> 近战  (Backport)
